spec/index.bs

```diff
@@ -231,25 +206,46 @@ interface mixin GPUObjectBase {
 };
 </script>
 
+{{GPUObjectBase}} has the following attributes:
+
 <dl dfn-type=attribute dfn-for=GPUObjectBase>
     : <dfn>label</dfn>
     ::
+        A label which can be used by development tools (such as error/warning messages,
+        browser developer tools, or platform debugging utilities) to identify the underlying
+        [=internal object=] to the developer.
+        It is not machine-readable.
```

Comment:
What does this mean?

Reply:
"It has no specified format, and therefore cannot be reliably machine-parsed."?